src/auro-checkbox-group.js

```diff
@@ -255,20 +252,26 @@ export class AuroCheckboxGroup extends LitElement {
    * @returns {void}
    */
   updated(changedProperties) {
-    if (this.disabled && changedProperties.has('disabled')) {
-      this.items.forEach((el) => {
-        el.disabled = this.disabled;
-      });
+    if (changedProperties.has('disabled') || changedProperties.has('validity')) {
+      this.handleCheckboxAttributes();
     }
 
     if (changedProperties.has('required')) {
-      this.items.forEach((el) => {
-        el.required = this.required;
-      });
+      if (this.required) {
+        this.setAttribute('aria-required', true);
+      } else {
+        this.removeAttribute('aria-required');
+      }
```

Comment:
I think this can and should be done in the constructor.

```js
/**
 * @private
 */
this.aria-required = this.required

aria-required: {
  type: Boolean,
  reflect: true
},
```